app/javascript/app/components/teaser/teaser-component.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+import Button from 'components/button';
+import Intro from 'components/intro';
+import Section from 'components/section';
+import introDark from 'styles/themes/intro/intro-dark.scss';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import buttonStyles from 'components/button/button-styles.scss';
+import styles from './teaser-styles.scss';
+
+class Teaser extends PureComponent {
+  // eslint-disable-line react/prefer-stateless-function
+  render() {
+    const { screenshot, title, description } = this.props;
+    return (
+      <div className={styles.wrapper}>
+        <Section className={styles.section}>
+          <div className={styles.column}>
+            <img
```

Comment:
I see a border button that doesn't exist in the design 
![image](https://user-images.githubusercontent.com/10500650/32099009-d6a17078-bb0e-11e7-8ae2-c12b95a9f95d.png)
It is very easy adding a margin negative in the img and a overflow hidden in the parent component